content/applications/sales/crm/performance/win_loss.rst

```diff
@@ -1,26 +1,100 @@
-=========================
-Check your Win/Loss Ratio
-=========================
+========================
+Create a win/loss report
+========================
 
-To see how well you are doing with your pipeline, take a look at
-the Win/Loss ratio.
+A win/loss ratio is the number of leads or opportunities a salesperson, sales team, or company have
+won and lost, often represented as a percentage of wins over losses. In the *CRM* app, win/loss
+ratios are available by creating a win/loss report.
 
-To access this report, go to your *Pipeline* view under the
-*Reporting* tab.
+To create this report, navigate to :menuselection:`CRM app --> Reporting --> Pipeline`.
 
-From there you can filter to which opportunities you wish to see, yours,
-the ones from your sales channel, your whole company, etc. You can then
-click on filter and check Won/Lost.
+.. image:: win_loss/reporting-tab-and-pipeline-view.png
+   :align: center
+   :alt: View of CRM page showing the Reporting tab with the Pipeline view highlighted.
 
-.. image:: win_loss/win_loss01.png
-    :align: center
+The :guilabel:`Pipeline` dashboard can be customized with various filters and group by options.
+Click the dropdown menu in the :guilabel:`Search...` bar, and consider the following common
+selections:
 
-You can also change the *Measures* to *Total Revenue*.
+#. Filters: :guilabel:`Won`.
+#. Filters: :guilabel:`Lost`.
 
-.. image:: win_loss/win_loss02.png
-    :align: center
+   - Optional: Remove or adjust the :guilabel:`Created On` filter to reflect the last 30 days, the
+     last quarter. By default, it reflects the entire year.
 
-You also have the ability to switch to a pie chart view.
+   - Optional: Show only leads that have reached the last stage of the sales pipeline (*Closed*) by
+     adding a custom filter for :menuselection:`Last Stage Update is >= DATE`.
+#. Group By: :guilabel:`Salesperson`.
+#. Group By: :menuselection:`Add Custom Group --> Active`.
 
-.. image:: win_loss/win_loss03.png
-    :align: center
+   - Optional: Group by :guilabel:`Sales Team` or :guilabel:`Company` instead of by
+     :guilabel:`Salesperson`.
+
+.. image:: win_loss/search-panel-filters-and-group-by-options.png
+   :align: center
+   :alt: View of CRM page showing the Search Panel open and the Won and Lost filters highlighted.
+
+After making the above selections, the :guilabel:`Search Panel` should read:
+:guilabel:`Created On: YYYY`, :guilabel:`Won or Lost`, :guilabel:`Salesperson > Active`. If so, the
```

Comment:
I don't know if we need to change this one.

When I open CRM in 17.0, I land at the Kanban view. I then open the Reporting menu and click Pipeline. That lands me at the Pipeline Analysis page that already has a bar graph open.

When I open the search bar dropdown and select the filters/group bys, that automatically changes the bar graph to reflect those selections. So really, all they need to do is click off the search bar dropdown (to make it disappear) and then they are viewing the bar graph by default.

I callout the chart icons later because I think they're worth knowing about as additional views, but they aren't relevant for just pulling the base report. Thoughts?